<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #292F36;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="IE_frame_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>600</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #495867;
border-style: solid;
border-width: 3px;
border-color: #EAEAEA;
padding: 4px;
border-radius: 4px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Arial&quot;;
border-style: none;
color: #EAEAEA;
background-color: none;</string>
     </property>
     <property name="text">
      <string>Insert and secure Ambu bag and ensure
 ventilation area is clear. When ready to 
start homing procedure, click &quot;Start&quot;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-20</y>
      <width>781</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 45pt &quot;Nimbus sans L&quot;;
background-color: none;
color: #EAEAEA;</string>
    </property>
    <property name="text">
     <string>Homing Procedure</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>761</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>125</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>125</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #495867;
border: 3px solid;
border-color: #EAEAEA;
padding: 2px;
font:36pt &quot;Nimbus sans L&quot;;
color: #EAEAEA;
border-radius: 4px;
</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="start_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>125</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>125</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #EAEAEA;
border: 3px solid;
border-color: #495867;
padding: 2px;
font:36pt &quot;Nimbus sans L&quot;;
color: #292F36;
border-radius: 4px;</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>WhiteVLogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
